### DISCUSSION TITLE
Copilot Bug in Robot Framework: Using [Tab] to Accept Suggestion Does NOT Work

### DISCUSSION THREAD

**Discussion by carterkunstek** [community/community#7480] [Projects and Issues] [answered]
I'm working in Visual Studio Code (v1.61.2) in .robot files with the extension "Robot Framework Language Server" by Robocorp (v0.28.0).  When working in these .robot files, Github Copilot provides suggestions just as expected and just as it does in any other file type.  However, when I hit [Tab] to accept a suggestion, the suggestion is NOT accepted, and it simply tabs over 3 spaces like a normal tab.  Therefore, the only way to accept a suggestion is to hover over it and click the "Accept [Tab]" button.

Copilot works fantastically in all other file types I've tried, but not in these .robot files with the provided robot extension.  

Is this happening to anyone else, and does anyone know of a solution to this bug?

****TL;DR****
[Tab] to accept suggestion does not work properly in .robot files in VScode with the extension  "Robot Framework Language Server".   Any ideas?

**Comment by dhirschfeld** [comment]
I'm seeing this in `terraform` files with VSCode Insiders 1.64.0

**Comment by rucindrea** [comment, marked as answer]
I had the same problem and noticed the Robot Framework Language server extensions adds a Tab keybinding:
![image](https://user-images.githubusercontent.com/3305864/150784959-f9677f0e-3f73-4063-8735-bdb1ef8f83b8.png)

I changed that to something else and now Copilot works again just fine.

**Reply by carterkunstek** [reply]
Thank you @dhirschfeld!  This fixed it for me.  Much appreciated!!

**Reply by dhirschfeld** [reply]
Thanks for sharing @rucindrea! That gives me an angle to investigate...
